Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -275,7 +275,7 @@ pcap_t* PcapLiveDevice::doOpen(const DeviceConfiguration& config)
 	{
 		PCPP_LOG_ERROR(pcap_geterr(pcap));
 		pcap_close(pcap);
-		pcap = NULL;
+		return NULL;

```

Comment:
@clementperon I guess this is the only fix that is related to the segfault, is that correct? 
I suggest we keep only that in this PR and move the rest of the changes (if they're needed) to a separate PR. Please let me know what you think

Reply:
done